docs/manual/docs/cmd/spo/listitem/listitem-record-declare.md

```diff
@@ -0,0 +1,58 @@
+# spo listitem record declare 
+
+Declares a list item as a record
+
+## Usage
+
+```sh
+spo listitem record declare [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --webUrl <webUrl>`|URL of the site where the list is located
+`-i, --id <id>`|ID of the list item to be declared as a record
+`-l, --listId [listId]`|ID of the list where the list item is located. Specify listId or listTitle but not both
+`-t, --listTitle [listTitle]`|Title of the list where the list item is located. Specify listId or listTitle but not both
+`-d, --date [date]`|Record declaration date
+`--verbose `|Runs command with verbose logging
+`--debug `| Runs command with debug logging
+
+!!! important
+    Before using this command, log in to a SharePoint Online site, using the [spo login](../login.md) command.
+
+## Remarks
+
+To declare an item as a record, you have to first log in to a SharePoint site using the [spo login](../login.md) command, eg. `spo login https://contoso.sharepoint.com`.
+
+## Examples
```

Comment:
Examples in the .md file should be the same as examples in the command's help

Reply:
Done - after triple checking I am 99.8% sure :)